### ISSUE TITLE
Cannot start NoVNC Docker Container on Ubuntu

### ISSUE THREAD

**Issue by jmewes** [rgardler/simdem#39]
# Observed result
After starting the `./scripts/run.sh` like mentioned in the [README](https://github.com/rgardler/simdem/blob/master/README.md) file, there is an error mentioning a missing manifest file.

```
docker: Error response from daemon: manifest for rgardler/simdem_novnc:0.7.4-dev not found.
```

# Expected result
Demo tutorial is available in web browser under the URL http://localhost:8080/?password=vncpassword

# Steps to reproduce
```
git clone https://github.com/rgardler/simdem.git
cd simdem
./scripts/run.sh
```

# Environment
- Ubuntu 16.04
- Docker version 17.06.0-ce, build 02c1d87

# Diagnostic information
Here is the full output after running the command.
```
$ ./scripts/run.sh 
Running rgardler/simdem_novnc:0.7.4-dev
Stopping and removing pre-existing containers
Error response from daemon: No such container: simdem_novnc
Error response from daemon: No such container: simdem_novnc
simdem_novnc_scripts
simdem_novnc_scripts
Creating scripts data container named simdem_novnc_scripts containing the scripts in /home/janux/src/misc/simdem/demo_scripts
bbd47f0ef8984b27f35245f46219606af71c284875d1e659460990c826d26f38
starting the simdem_novnc container in mode tutorial
Unable to find image 'rgardler/simdem_novnc:0.7.4-dev' locally
docker: Error response from daemon: manifest for rgardler/simdem_novnc:0.7.4-dev not found.
See 'docker run --help'.
```

**Comment by jmewes** [comment]
# Workaround
The problem is that the image `` doesn't exist on Docker Hub.
```
$ docker search rgardler | grep -i novnc
rgardler/simdem_novnc                    SimDem: documents that are tutorials, demo...   0                    [OK]
```
After pulling the image `rgardler/simdem_novnc` and tagging it with `0.7.4-dev`, the noVNC URL is available.
```
docker pull rgardler/simdem_novnc
docker tag rgardler/simdem_novnc:latest rgardler/simdem_novnc:0.7.4-dev
./scripts/run.sh
firefox http://localhost:8080/?password=vncpassword
```

**Comment by rgardler** [comment]
Thanks for reporting.

tl;dr

`git pull upstream`
`./scripts/run.sh`

Explanation:

SimDem pulls the version number to run from the config.py file - that is it will always run the most recent developer code. We don't push in development code to Docker Hub. I had expected that people would do `build.sh` before trying `run.sh` - not a good assumption.

I've added this step to the build script. This will make the first execution of 'run.sh' slower but subsequent ones will be nice and quick.

Thanks for reporting.

**Comment by jmewes** [comment]
Thanks for fixing this.
